tempto-core/src/main/java/com/teradata/tempto/sql/SqlContexts.java

```diff
@@ -41,6 +42,11 @@ public static ViewContextProvider createViewAs(String selectSql)
         return new ViewContextProvider(viewName, selectSql, defaultQueryExecutor());
     }
 
+    public static ViewContextProvider createViewAs(String viewName, String selectSql, QueryExecutor queryExecutor)
+    {
+        return new ViewContextProvider(viewName, selectSql, queryExecutor);
```

Comment:
Cool, thanks. LGTM. I will post this in the testing framework chat so that someone else can look at it as well.